R/brdg_hierarchical.R

```diff
@@ -61,11 +61,10 @@ brdg_hierarchical <- function(cards,
     cards <- cards |>
       mutate(
         variable_level = ifelse(
-          .data$variable == "..ard_hierarchical_overall..", label[["overall"]], .data$variable_level
```

Comment:
FYI I think it'll be easier in the long run to retain the name from the ARD output, and the name overall may be too common and cause some name conflicts in the future